internal/operator-controller/rukpak/render/render.go

```diff
@@ -111,7 +124,7 @@ type BundleRenderer struct {
 	ResourceGenerators []ResourceGenerator
 }
 
-func (r BundleRenderer) Render(rv1 bundle.RegistryV1, installNamespace string, opts ...Option) ([]client.Object, error) {
+func (r BundleRenderer) Render(rv1 bundle.RegistryV1, opts ...Option) ([]client.Object, error) {
```

Comment:
Let's not make install namespace optional here. We will _always_ need an install namespace. So, I'd revert the changes to this package.